video/out/vo_sixel.c

```diff
@@ -0,0 +1,483 @@
+/*
+ * video output driver for libsixel
+ * Based on the code for FFMPEG Sixel output driver by Hayaki Saito
```

Comment:
Please add a link to the original file, as well as explicit copyright of the original author exactly as it appears at the original file.

Also, you changed the license from LGPL (the original file and new files in mpv) to GPL - which is stricter than LGPL.

Please change the license clause to LGPL - you can copy it from `vo_tct.c`.

Reply:
Done